docs/src/main/sphinx/release/release-307.rst

```diff
@@ -14,76 +14,76 @@ General changes
 * Add support for outer joins involving lateral derived tables (i.e., ``LATERAL``). (:issue:`390`)
 * Add support for setting table comments via the :doc:`/sql/comment` syntax. (:issue:`200`)
 
-Web UI changes
---------------
+Web UI
+------
 
 * Allow UI to work when opened as ``/ui`` (no trailing slash). (:issue:`500`)
 
-Security changes
-----------------
+Security
+--------
 
 * Make query result and cancellation URIs secure. Previously, an authenticated
   user could potentially steal the result data of any running query. (:issue:`561`)
 
-Server RPM changes
-------------------
+Server RPM
+----------
 
 * Prevent JVM from allocating large amounts of native memory. The new configuration is applied
   automatically when Presto is installed from RPM. When Presto is installed another way, or when
   you provide your own ``jvm.config``, we recommend adding ``-Djdk.nio.maxCachedBufferSize=2000000``
   to your ``jvm.config``. See :doc:`/installation/deployment` for details. (:issue:`542`)
 
-CLI changes
------------
+CLI
+---
 
 * Always abort query in batch mode when CLI is killed. (:issue:`508`, :issue:`580`)
 
-JDBC driver changes
--------------------
+JDBC driver
+-----------
 
 * Abort query synchronously when the ``ResultSet`` is closed or when the
   ``Statement`` is cancelled. Previously, the abort was sent in the background,
   allowing the JVM to exit before the abort was received by the server. (:issue:`580`)
 
-Hive connector changes
-----------------------
+Hive connector
+--------------
 
 * Add safety checks for Hive bucketing version. Hive 3.0 introduced a new
   bucketing version that uses an incompatible hash function. The Hive connector
   will treat such tables as not bucketed when reading and disallows writing. (:issue:`512`)
 * Add support for setting table comments via the :doc:`/sql/comment` syntax. (:issue:`200`)
 
-MySQL connector changes
------------------------
+MySQL connector
+---------------
 
 See `Base-JDBC connector library changes <#base-jdbc-connector-library-changes>`__.
```

Comment:
these links need to be updated too

Reply:
Ah, I missed this one. I'll update it to the newer "Other connector" language.